api/src/routes/edit.ts

```diff
@@ -130,10 +130,9 @@ edit.get("/object/modelsdatastreams/:pid", requireToken, pidSanitizer, async fun
         res.status(500).send(error.message);
     }
 });
-edit.post("/object/:pid", requireToken, pidSanitizer, uploadFile);
-edit.get("/object/children", requireToken, getChildren);
-edit.get("/object/children/:pid", requireToken, pidSanitizer, getChildren);
-edit.get("/object/details/:pid", requireToken, pidSanitizer, async function (req, res) {
+edit.get("/treeTop", requireToken, getChildren);
```

Comment:
is this supposed to be hierarchy tree? or root of the tree?

Reply:
It's just listing the children of the root of the tree -- top-level PIDs. I'm not attached to calling it treeTop. Could be treeRoot, or any better name you care to suggest.

Reply:
Maybe topLevelTree?

Reply:
Thanks for the suggestion. Since this doesn't really directly relate to the entire tree, I opted to change it to topLevelObjects -- I think that's the most descriptive name. I've just pushed up a commit. What do you think?

Reply:
I think that's a closer description.